Tfe updates (#2215)
hashicorp/terraform-website: content/enterprise/system-overview/capacity.mdx

The settings for per-run memory and concurrency are available in the dashboard on port 8800, on the Settings page, under the Capacity section. They can also be set via
the [application settings JSON file when using the automated install procedure](/enterprise/install/automated/automating-the-installer#available-settings).

## Increasing Concurrency

> To increase the number of concurrent runs, adjust the `capacity_concurrency` setting. Note that this setting is not limited by

Reviewer: This is a nitpick, but I think we shouldn't start with "Note that" unless we're going to put something in a Note block. Otherwise, we can just make it a statement. 

```suggestion
To increase the number of concurrent runs, adjust the `capacity_concurrency` setting. This setting is not limited by
```

system checks; it depends on the operator to provide enough memory to the system to accommodate the requested

> concurrent capacity. For instance, if `capacity_concurrency` is set to `100`, the instance would require, at a minimum,
> 52 GB of memory just for Terraform runs, if the worker memory was set to 512. The rest of the TFE application requires 4 GB of memory at a minimum, on top of the Operating System requirements.

Reviewer: ```suggestion
concurrent capacity. For example, if `capacity_concurrency` is set to `100` and the worker memory is set to 512, the instance would require a minimum of 52 GB of memory just for Terraform runs. The rest of the TFE application requires a minimum of 4 GB of memory in addition to the Operating System requirements.
```

If the memory limit is adjusted to 1024 MB with the default capacity of 10, the instance would require, at a minimum,
10 GB of memory reserved for Terraform runs.

## Limiting CPU cores

> The default setting of `0` allows a Terraform run to utilize all available CPU cores on the TFE instance. You can adjust this setting by inputting a specific limit to the number of CPU cores the Terraform runs will have access to. 

Reviewer: Thinking of this, could we tell folks where to adjust this? What's the name of the attribute or whatever that they need to set. I'm thinking we say "You can also set a value for `attribute_name` to specify a limit for the number of CPU cores Terraform runs can access." Or something similar. Thoughts?

```suggestion
The default setting of `0` allows a Terraform run to use all available CPU cores on the TFE instance. You can also input a limit for the number of CPU cores Terraform runs can access. 
```

Because of the amount of churn caused by container creation as well as Terraform state management,
highly concurrent setups will begin pushing hard on disk I/O. In cloud environments like AWS that limit disk
I/O to IOPS that are credited per disk, it's important to provision a minimum number to prevent I/O related
stalls. Low disk I/O can create significant performance issues. 

This resource is harder to predict than memory or CPU usage because it varies per Terraform module,

> but in general we recommend a minimum of 50 IOPS per concurrent Terraform run. So if an instance is configured for 10 concurrent runs, the disk should have 500 IOPS allocated. For reference, on AWS, an EBS volume

Reviewer: ```suggestion
but we generally recommend a minimum of 50 IOPS per concurrent Terraform run. So if an instance is configured for 10 concurrent runs, the disk should have 500 IOPS allocated. For reference, on AWS, an EBS volume
```

with an allocated size of 250 GB comes with a steady state of 750 IOPS.

> It is recommended to use a disk with a minimum of 500 IOPS, but high load systems should consider increasing this significantly. A production instance, with a consistantly high level of utilization, and a concurrency of 10, should have a disk with around 3,000 IOPS, ideally. Internal testing has shown performance increases with additional IOPs up to 8,000. Scaling the disk beyond 8,000 IOPs does not significantly improve performance.

Reviewer: ```suggestion
We recommend using a disk with a minimum of 500 IOPS, but high load systems should consider increasing this significantly. For example, a production instance with a consistently high level of utilization and a concurrency of 10 should ideally have a disk with about 3,000 IOPS. Internal testing has shown performance increases with additional IOPs up to 8,000. Scaling the disk beyond 8,000 IOPs does not significantly improve performance.
```

Because of the amount of churn caused by container creation as well as Terraform state management,
highly concurrent setups will begin pushing hard on disk I/O. In cloud environments like AWS that limit disk
I/O to IOPS that are credited per disk, it's important to provision a minimum number to prevent I/O related
stalls. Low disk I/O can create significant performance issues. 

This resource is harder to predict than memory or CPU usage because it varies per Terraform module,

> but in general we suggest a minimum of 50 IOPS per concurrent Terraform run. So if an instance is configured for
> 10 concurrent runs, the disk should have 500 IOPS allocated. For reference, on AWS, an EBS volume

Reviewer: ```suggestion
but in general we recommend a minimum of 50 IOPS per concurrent Terraform run. So if an instance is configured for 10 concurrent runs, the disk should have 500 IOPS allocated. For reference, on AWS, an EBS volume
```

with an allocated size of 250 GB comes with a steady state of 750 IOPS.

> It is recommended to use a disk with a minimum of 500 IOPS, but high load systems should consider increasing this significantly. A production instance, with a consistantly high level of utilization, and a concurrency of 10, should have a disk with around 3,000 IOPS, ideally. 

Reviewer: I like that. Also I think we say "we recommend" rather than "it is recommended"  One nit is just I think I'd say "Scaling the disk beyond 8,000 IOPs does not significantly improve performance."

I think add it and I'll do a copyedit pass :-)
